sedaro/src/sedaro/branches/scenario_branch/sim_client.py

```diff
@@ -118,6 +119,24 @@ def set_nested(results):
         nested[k] = (results[k][0], {kspl: set_numeric_as_list(__set_nested(results[k][1][kspl]))})
     return nested
 
+class FastFetcherResponse:
```

Comment:
Could we add a getattr fallback to this that proxies `response`? Something like:

```py
def __getattr__(self, key): return self.response[key]
```

This will allow us to access more attributes from the response in the future if we need to. Note that you'll need to add `response` as a instance attribute in the constructor.